.github/workflows/CodeQuality.yaml

```
@@ -66,10 +72,11 @@ jobs:
 
   sonarqube:
     runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
+    needs: unit-test 
+      - name: Download coverage report
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 76: mapping values are not allowed in this context

(syntax-check)